app/assets/javascripts/discourse/templates/components/user-card-contents.hbs

```diff
@@ -1,223 +1,263 @@
 {{#if visible}}
   <div class="card-content">
+    {{#if loading}}
+      <div class="card-row first-row">
+        <div class="user-card-avatar">
+          <div class="card-avatar-placeholder animated-placeholder"></div>
+        </div>
+      </div>
 
-    <div class="card-row first-row">
-      <div class="user-card-avatar">
-        {{#if user.profile_hidden}}
-          <span class="card-huge-avatar">{{bound-avatar user "huge"}}</span>
-        {{else}}
-          <a href="{{user.path}}" {{action "showUser" user}} class="card-huge-avatar">{{bound-avatar user "huge"}}</a>
-        {{/if}}
-        {{#if user.primary_group_name}}
-          {{avatar-flair
-          flairURL=user.primary_group_flair_url
-          flairBgColor=user.primary_group_flair_bg_color
-          flairColor=user.primary_group_flair_color
-          groupName=user.primary_group_name}}
-        {{/if}}
-        {{plugin-outlet name="user-card-avatar-flair" args=(hash user=user) tagName='div'}}
+      <div class="card-row second-row">
+        <div class="animated-placeholder"></div>
+      </div>
+      <div class="card-row third-row">
+        <div class="animated-placeholder"></div>
       </div>
-      <div class="names">
-        <h1 class="{{staff}} {{newUser}} {{if nameFirst "full-name" "username"}}">
+      <div class="card-row fourth-row">
+        <div class="animated-placeholder"></div>
+      </div>
+      <div class="card-row sixth-row">
+        <div class="animated-placeholder"></div>
+      </div>
+    {{else}}
+      <div class="card-row first-row">
+        <div class="user-card-avatar">
           {{#if user.profile_hidden}}
-            <span class="name-username-wrapper">
-              {{if nameFirst user.name (format-username username)}}
-            </span>
+            <span class="card-huge-avatar">{{bound-avatar user "huge"}}</span>
           {{else}}
-            <a href="{{user.path}}" {{action "showUser" user}} class='user-profile-link'>
+            {{#if loading}}
```

Comment:
It does not make sense anymore to keep this check here because we already checked for this on line 3 and it is ensured loading will always be falsey.

Reply:
Ouh yeah, forgot to remove that ._.